recipes/azureml/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "azureml" %}
+{% set version = "0.2.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/azureml-{{ version }}.zip
+  sha256: 1b48afa1306d1f32133748585a3e5a6793557780105f4e51c2a978070dd432f1
+
+build:
+  skip: True  # [win or osx or py<36]
```

Comment:
Any reason for not making this one noarch? If it really is a Linux only package you can do OS-noarch with https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-packages-with-os-specific-dependencies

Reply:
Let me redo it to see if it can be done for all OS. Thanks in advance

Reply:
done 🫡

Reply:
do I need to ping the team again?